gatsby-site/gatsby-node.js

```diff
@@ -0,0 +1,32 @@
+const path = require("path");
```

Comment:
Is there something that changed in this file? It looks the same, but it says the whole thing is new?

Reply:
I don't think I changed anything here. Weird that it is showing up as a new file. This was created when I installed Remark.